docs/release/trg-0/trg-0-template.md

```diff
@@ -16,8 +16,15 @@ Proposed release date: "mandatory after": 19th of May 2023
 
 ## Why
 
-Describe in this section why the TRG exist. For example because it's a legal requirement or it makes it much easier for others to use or it's an industry standard.
+Provide reasoning behind the release guideline. Why does Eclipse Tractus-X align on this quality criteria.
```

Comment:
```suggestion
This section must provide the reason behind the release guideline and a description why the Eclipse Tractus-X community have to align on this quality criteria.
```

Reply:
Slightly adapted your suggestion. What do you think? [46c3bea](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/871/commits/46c3bea55e19dc1f4dabb2012c7042bdd493f428)